packages/http-client/src/ceramic-http-client.ts

```diff
@@ -46,9 +40,8 @@ export const DEFAULT_CLIENT_CONFIG: CeramicClientConfig = {
  */
 export interface CeramicClientConfig {
```

Comment:
Can you add tsdoc to each of the config params here?

Reply:
k